Dockerfile

```diff
@@ -1,70 +1,43 @@
-# build frontend
-FROM node:lts-buster AS fe-builder
+FROM golang:1.17.7-alpine as cloudreve_builder
 
-COPY ./assets /assets
 
-WORKDIR /assets
+# install dependencies and build tools
+RUN apk update && apk add --no-cache wget curl git yarn build-base gcc abuild binutils binutils-doc gcc-doc
 
-# If encountered problems like JavaScript heap out of memory, please uncomment the following options
-ENV NODE_OPTIONS --max_old_space_size=4096
+WORKDIR /cloudreve_builder
+RUN git clone --recurse-submodules https://github.com/cloudreve/Cloudreve.git
 
-# yarn repo connection is unstable, adjust the network timeout to 10 min.
-RUN set -ex \
-    && yarn install --network-timeout 600000 \
-    && yarn run build
+# build frontend
+WORKDIR /cloudreve_builder/Cloudreve/assets
+RUN yarn install --network-timeout 1000000
+RUN yarn run build
 
 # build backend
-FROM golang:1.17.7-alpine3.12 AS be-builder
-
-ENV GO111MODULE on
-
-COPY . /go/src/github.com/cloudreve/Cloudreve/v3
-COPY --from=fe-builder /assets/build/ /go/src/github.com/cloudreve/Cloudreve/v3/assets/build/
-
-WORKDIR /go/src/github.com/cloudreve/Cloudreve/v3
-
-RUN set -ex \
-    && apk upgrade \
-    && apk add gcc libc-dev git \
+WORKDIR /cloudreve_builder/Cloudreve
+RUN go get github.com/rakyll/statik \
+    && statik -src=assets/build/ -include=*.html,*.js,*.json,*.css,*.png,*.svg,*.ico -f \
+    && tag_name=$(git describe --tags) \
     && export COMMIT_SHA=$(git rev-parse --short HEAD) \
-    && export VERSION=$(git describe --tags) \
-    && go install -ldflags "-X 'github.com/cloudreve/Cloudreve/v3/pkg/conf.BackendVersion=${VERSION}' \
-                            -X 'github.com/cloudreve/Cloudreve/v3/pkg/conf.LastCommit=${COMMIT_SHA}'\
-                            -w -s"
-
-# build final image
-FROM alpine:3.12 AS dist
+    && go build -a -o cloudreve -ldflags " -X 'github.com/HFO4/cloudreve/pkg/conf.BackendVersion=$tag_name' -X 'github.com/HFO4/cloudreve/pkg/conf.LastCommit=$COMMIT_SHA'"
```

Comment:
Should be cloudreve/Cloudreve

Reply:
Good catch! I'll create another PR for fixing this, thanks!